build.xml

```diff
@@ -541,8 +543,10 @@
 
           <dependency groupId="org.yaml" artifactId="snakeyaml" version="1.11"/>
           <dependency groupId="junit" artifactId="junit" version="4.12" />
+          <dependency groupId="org.mockito" artifactId="mockito-core" version="3.2.4" />
           <dependency groupId="org.quicktheories" artifactId="quicktheories" version="0.25" />
           <dependency groupId="com.google.code.java-allocation-instrumenter" artifactId="java-allocation-instrumenter" version="${allocation-instrumenter.version}" />
+          <dependency groupId="org.apache.cassandra" artifactId="dtest-api" version="0.0.1-SNAPSHOT" />
```

Comment:
snapshots are unsafe since a change to dtest now breaks cassandra...  should only rely on release.

Reply:
I will release a 0.0.1 before commit